form/src/main/java/com/ritense/form/service/impl/FormIoFormLoaderService.java

```diff
@@ -56,4 +56,13 @@ public Optional<JsonNode> getFormDefinitionByNamePreFilled(final String formDefi
             );
     }
 
-}
\ No newline at end of file
+    @Override
+    public Optional<JsonNode> getFormDefinitionByNamePreFilled(final String formDefinitionName, final JsonNode content) {
```

Comment:
I don't see this making use of the document for prefilling. Any way we can incorporate that here?

Reply:
Thats outside the scope of the story. But I added it anyway